### ISSUE TITLE
Training for 10 hours with warmup without results

### ISSUE THREAD

**Issue by titanbender** [experiencor/basic-yolo-keras#200]
My goal is to use the basic yolo keras for extracting areas of interest. Before applying the model on my custom dataset that I prepared, I wanted to test it on the raccoon example dataset. 

AWS wouldn't allow me to install openCV on python 2.7, so I updated all the code to python 3 following the comments in the thread (print statements, xrange, etc.)

I've followed the guide thoroughly. Warmed up with 3 epochs - saved to separate weight file - and let the model train for 10 hours (100 epocs) on AWS by using the pretrained file. However, the errors doesn't seem to improve at all. In fact, it wasn't able to detect a raccoon after all that training. 

Been reading all the other issues posts - and I can't seem to find what I'm doing wrong. Has anyone experienced the same? 

<img width="959" alt="screen shot 2018-03-22 at 9 00 28 am" src="https://user-images.githubusercontent.com/12022636/37759088-e16a24e8-2db2-11e8-9bab-439a0be594bb.png">
